__tests__/Utilities.test.js

```diff
@@ -110,6 +112,58 @@ describe('Utilities', () => {
 
   })
 
+  describe('generateTitle()', () => {
+
+    it('returns a label', () => {
```

Comment:
since this returns diff things in diff circumstances, having the description say something about what circumstances cause it to return XXX might be helpful.  Or maybe it's enough to just look at the data ... but the tests read a little weird (it returns A.  it returns B.  it returns C.)  This could be helpful for maintenance but it's kinda picky so ok if you disagree.

Reply:
This is being removed.